src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -87,6 +90,23 @@ private void WriteRunToErrorList(Run run, string logFilePath, Solution solution)
 
             var projectNameCache = new ProjectNameCache(solution);
 
+            if (run.Files != null)
+            {
+                foreach (var file in run.Files)
+                {
+                    var fileHash = file.Value.Hashes?.First(x => x.Algorithm == AlgorithmKind.Sha256)?.Value;
+                    var contents = file.Value.Contents;
+                    var key = new Uri(file.Key);
+                    if (fileHash != null && contents != null)
+                    {
+                        var fileDetails = new FileDetailsModel(fileHash, file.Value.Contents);
```

Comment:
>FileDetailsModel [](start = 46, length = 16)

why doesn't this class accept a FileData instance in the ctor? Seems like breaking apart members will just leave us in a position to keep updating this code.